protocol/avdl/chat1/common.avdl

```diff
@@ -187,6 +187,12 @@ protocol common {
     bytes k; // Verifying key
   }
 
+  record SignEncryptedData {
+    int   v; // version = 1
+    bytes b; // signEncryptedData (output of signencrypt.SealWhole)
+    bytes n; // nonce
```

Comment:
`SignatureInfo` above includes the verify key inline there. Did we think about including it inline here? I could go either way, since the TLFName + KeyGeneration info will be outboard anyway.